src/dotnet/dotnet.csproj

```diff
@@ -1,6 +1,5 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Visual Studio seemed to have thought it is better to remove them. Since it will do it for anyone who is using it and we are just introducing the logic to make people not touch csprojej manually I think it is better to just let it..